app/page.tsx

```diff
@@ -1,5 +1,7 @@
 import ApplicationTable from '@/components/ApplicationTable'
 import prisma from '@/db'
+import { Role } from '@prisma/client'
+import { log } from 'console'
```

Comment:
```suggestion
```

Reply:
removed in 3993a3331e005e1ae1fa3b79a364ca8ee3c2f24f